share/man/man1/brew.1

```diff
@@ -554,10 +554,6 @@ HOMEBREW_BROWSER
 If set, uses this setting as the browser when opening project homepages, instead of the OS default browser\.
 .
 .TP
-HOMEBREW_BUILD_FROM_SOURCE
```

Comment:
I used to *always* build from source. Now I *often* build from source, either for performance reasons, or to avoid bottling pitfalls such as https://github.com/Homebrew/homebrew/issues/46553. Another situation we often encounter in homebrew-science is that `hdf5` will get bumped to a new version, and a plethora of dependents will need to be bumped in the appropriate order (so the bot doesn't time out and builds don't fail). Being able to build from source is quite crucial.